lib/react/renderer.rb

```diff
@@ -22,9 +23,9 @@ def self.setup!(react_js, components_js, args={})
       @@pool = ConnectionPool.new(default_pool_options.merge(args)) { self.new }
     end
 
-    def self.render(component, args={})
+    def self.render(component, args={}, replay_console=false)
```

Comment:
A config would be good by me, then you could just turn it always-on in development and always-off in production!

As for mixing args, it's alright in my book by way of precedent in the [view helper](https://github.com/reactjs/react-rails/blob/master/lib/react/rails/view_helper.rb#L20-L21)